### ISSUE TITLE
[v2.13.0] NPE on SplitEventTask.onPostExecution

### ISSUE THREAD

**Issue by ekosuhariyadi** [splitio/android-client#432]
Hi, we are using android-client v2.13.0 and recently we found crashes in Firebase Crashlytics 
(85% happen when app in background)

the stack trace
```
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'void io.split.android.client.events.SplitEventTask.onPostExecution(io.split.android.client.SplitClient)' on a null object reference
       at io.split.android.client.events.executors.SplitEventExecutorWithClient$1.doInBackground(SplitEventExecutorWithClient.java:39)
       at io.split.android.client.events.executors.SplitEventExecutorWithClient$1.doInBackground(SplitEventExecutorWithClient.java:28)
```

We also check on the code
```
public SplitEventExecutorWithClient(SplitEventTask task, SplitClient client) {

        super(task);
        checkNotNull(task);
        _sclient = checkNotNull(client);
    }

    public void execute(){

        _asyncTansk = new AsyncTask<SplitClient, Void, SplitClient>() {

            @Override
            protected SplitClient doInBackground(SplitClient... splitClients) {

                if (splitClients.length > 0) {

                    SplitClient client = checkNotNull(splitClients[0]);

                    //BACKGROUND POST EXECUTION
                    try {
                        _task.onPostExecution(client);
```
but couldn't find a possibility of `_task` field being null

Thank you

**Comment by ekosuhariyadi** [comment]
Sure, thanks

**Comment by JaredBanyard** [comment]
@ekosuhariyadi @agustinona Was this issue resolved in 2.13.1?

**Comment by agustinona** [comment]
@JaredBanyard We haven't so far been able to reproduce this issue. I have reached out to you through the support ticket you just opened.

**Comment by ekosuhariyadi** [comment]
> @ekosuhariyadi @agustinona Was this issue resolved in 2.13.1?

sorry for the late reply, we integrate v2.13.1 in our recent release. Unfortunately, the crash is increasing than v2.13.0
